src/main/java/frc/robot/commands/Drivetrain/ClosedLoopDrive.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Drivetrain;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.RobotPreferences.DrivetrainPrefs;
+import frc.robot.subsystems.Drivetrain;
+
+public class ClosedLoopDrive extends CommandBase {
+  Drivetrain drivetrain;
+
+  double speed;
+  double turn;
+
+  double previousSpeed;
+  boolean isSpeedPositive;
+  boolean isSpeedIncreasing;
+
+  /** Creates a new ClosedLoopDrive. */
+  public ClosedLoopDrive(Drivetrain sub_drivetrain) {
+    drivetrain = sub_drivetrain;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(drivetrain);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    previousSpeed = 0;
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    // Read direction of the joystick
+    speed = RobotContainer.DriverStick.getArcadeMove();
+
+    if (speed > 0) {
```

Comment:
FYI, you could have just done
isSpeedPositive = (speed>0)
isSpeedIncreasing = (speed-previousSpeed)

Reply:
I copied this from `Drive.java` and imo it's not worth changing, but thanks for the note.